backend/apps/sandbox/views.py

```
@@ -582,3 +611,76 @@ def _normalize(text):
+from .models import LicenseScenario, DependencyDiff, LicenseAttempt
+from .serializers import LicenseScenarioSerializer, LicenseAttemptSerializer
+
+
+class LicenseScenarioViewSet(viewsets.ReadOnlyModelViewSet):
+    """
+    Viewset for License Scenarios (Dependency Detective).
+    """
+
+    queryset = LicenseScenario.objects.all().order_by("-created_at")
+    serializer_class = LicenseScenarioSerializer
+    permission_classes = [permissions.AllowAny]
```

Ruff (0.15.21)

[warning] Mutable default value for class attribute

(RUF012)